hyperjumptech/monika
# Feat(status): [450] tweet status

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
add link to post current status notification body to twitter

## How did you implement / how did you fix it  
1. create tweet message of current status
2. add message to twitter text link
3. add to status notification body

## How to test  
1. create monika config with email notification
2. run monika with `monika --status-notification "*/1 * * * *"`
3. check on mail status notification, there should be a link to twitter.
